armonik/ingress-configmap.tf

```diff
@@ -65,19 +65,35 @@ server {
     location = /admin/fr {
         rewrite ^ $scheme://$http_host/admin/fr/;
     }
+    # Deprecated, must be removed in a new version. Keeped for retrocompatibility
     location = /old-admin {
-        rewrite ^ $scheme://$http_host/old-admin/ permanent;
+        rewrite ^ $scheme://$http_host/admin-0.8/ permanent;
+    }
+    # Deprecated, must be removed in a new version
+    location = /admin-0.8 {
+        rewrite ^ $scheme://$http_host/admin-0.8/ permanent;
+    }
+    # Deprecated, must be removed in a new version
+    location = /admin-0.9 {
+        rewrite ^ $scheme://$http_host/admin-0.9/ permanent;
     }
```

Comment:
I think you would also need the language redirection for 0.9 as for `admin`.

Reply:
Yes, I thinks it could be useful.